### DISCUSSION TITLE
TinyMCE 6.7: Accessibility improvements, plugin enhancements and more

### DISCUSSION THREAD

**Discussion by mrinasugosh** [tinymce/tinymce#9044] [Announcements]
## TinyMCE 6.7 Release Notes - Overview
We are excited to announce the release of TinyMCE 6.7! Following the release of [TinyMCE’s AI Assistant plugin ](https://tiny.cloud/tinymce/features/ai-integration/?utm_campaign=devrel_github_discussions&utm_source=github&utm_medium=blog)in mid 2023, our team decided to refocus on a number of accessibility-related requests from customers, plugin enhancements, a paste issue, and some bugs in the backlog.

## What's New and Improved

1. [Advanced Templates ](https://www.tiny.cloud/tinymce/features/advanced-templates/?utm_campaign=devrel_github_discussions&utm_source=github&utm_medium=blog)now supports the {{mce-cursor}} tag to place cursor in templates
2. [Table of Contents](https://www.tiny.cloud/tinymce/features/table-of-contents/?utm_campaign=devrel_github_discussions&utm_source=github&utm_medium=blog) has new formatting options like ordered lists and header toggle
3. [Accessibility](https://www.tiny.cloud/tinymce/features/accessibility-checker/?utm_campaign=devrel_github_discussions&utm_source=github&utm_medium=blog) improvements:
 - More descriptive "More" toolbar button for screen readers
 - Clearer table grid announcements
 - Visible keyboard shortcut cue for accessing Help
4. [PowerPaste](https://www.tiny.cloud/tinymce/features/powerpaste/?utm_campaign=devrel_github_discussions&utm_source=github&utm_medium=blog) now strips comments when pasting from MS Word to improve security
5. Bug Fixes: TinyMCE 6.7 includes fixes for 21 bugs reported by the community. See the [changelog](https://github.com/tinymce/tinymce/blob/develop/modules/tinymce/CHANGELOG.md) for details.

## How to Get Access
 - If you're already on TinyMCE 6, upgrading is simple:
    - Cloud users will receive 6.7 automatically
    - Self-hosted users should update to the latest version
 - If you're still on TinyMCE 4 or 5, now is a great time to [upgrade](https://www.tiny.cloud/docs/tinymce/6/migration-from-5x/?utm_campaign=devrel_github_discussions&utm_source=github&utm_medium=blog) to 6.7. TinyMCE 5 reached end of support in April 2023.
 - If the premium plugins mentioned are not available in your current plan, [contact our Customer Success team](https://www.tiny.cloud/contact/?utm_campaign=devrel_github_discussions&utm_source=github&utm_medium=blog) about upgrading your account. You can also [sign up for a free 14-day trial](https://www.tiny.cloud/get-tiny/?utm_campaign=devrel_github_discussions&utm_source=github&utm_medium=blog) of TinyMCE premium plugins.
 
We value your feedback and suggestions. Let us know what features you would like to see next by submitting your feature requests or voting on existing features on the public [TinyMCE Roadmap](https://www.tiny.cloud/roadmap/?utm_campaign=devrel_github_discussions&utm_source=github&utm_medium=blog).

## Useful Links:
[Release Notes](https://www.tiny.cloud/docs/tinymce/6/6.7-release-notes/?utm_campaign=devrel_github_discussions&utm_source=github&utm_medium=blog)
[Changelog](https://github.com/tinymce/tinymce/blob/develop/modules/tinymce/CHANGELOG.md)
[Documentation](https://www.tiny.cloud/docs/tinymce/6/introduction-to-tinymce/?utm_campaign=devrel_github_discussions&utm_source=github&utm_medium=blog)
[Download](https://www.tiny.cloud/get-tiny/?utm_campaign=devrel_github_discussions&utm_source=github&utm_medium=blog)

Thank you for choosing TinyMCE!

-The Tiny Team

**Comment by vanillajonathan** [comment]
The Advanced Template plugin is very nice, but unfortunately now the old Template plugin is marked as deprecated.

I would like to ask if the decision to mark the Template plugin as deprecated could be reconsidered?

If it is because the Template plugin is considered so powerful that it competes with the Advanced Template plugin, then I would like to propose a very basic template plugin (without any advanced features such as preview, dynamic data, replacement, merge tags, etc), just a very simple one to be able to insert simple static snippets such as `<div class="callout">Hello</div>`.

**Reply by mrinasugosh** [reply]
@vanillajonathan We wanted to make the template plugin premium, but we cannot remove something immediately unless we do a major update, so we had to create an another plugin which we called advanced template. This is why for a while we had two plugins.
If a very basic template feature is required for your project, we would recommend that you build one using the APIs.

**Reply by vanillajonathan** [reply]
Alright, makes sense.

I would like to request for The Tiny Team to consider providing a small, simple template plugin with just the bare minimum for the hobbyist and personal users of TinyMCE. I am sure the Advanced Template plugin is great for enterprise customers, it is a very good plugin with lots of nice functionality, but I think should be a simple template plugin for the hobbyists with non-commercial pets projects.
